cmd/main.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
Nadie ha pedido un main, ni debéis ponerlo.

Reply:
Lo elimino.